mage_ai/api/resources/BlockRunResource.py

```diff
@@ -102,4 +102,21 @@ def collection(self, query_arg, meta, user, **kwargs):
                 filter(c.pipeline_uuid == pipeline_uuid)
             )
 
-        return query.order_by(a.created_at.desc(), a.completed_at.desc())
+        order_by_arg = query_arg.get('order_by', [None])
```

Comment:
Could you add comments to this code? What are supported values for `order_by`?

Reply:
And what do exceptions look like when `order_by` is in the wrong format?

Reply:
> Could you add comments to this code? What are supported values for order_by?
Added a comment.

> And what do exceptions look like when order_by is in the wrong format?
It'll look like the attached screenshot.
![image](https://github.com/mage-ai/mage-ai/assets/78053898/a8364872-e01b-451e-8613-e9f6efc0c1b0)